box-share-sdk/src/main/java/com/box/androidsdk/share/vm/BaseShareVM.java

```diff
@@ -66,4 +67,21 @@ public void fetchItemInfoFromRemote(BoxItem item) {
     public LiveData<PresenterData<BoxItem>> getItemInfo() {
         return mItemInfo;
     }
+
+
+    /**
+     * Returns user id of the current user.
+     * @return user id of the current user
+     */
+    public String getUserId() {
+        return mShareRepo.getUserId();
```

Comment:
This seems to break the observer pattern, why can't this be a LD of userId?

Reply:
we will not be observing the changes in userid.